src/FSharp.AWS.DynamoDB/Expression/ConditionalExpr.fs

```diff
@@ -515,11 +585,10 @@ type ConditionalExpression with
         let values = aw.Values |> Seq.map (fun kv -> kv.Key, kv.Value.Print()) |> Seq.toList
         expr, names, values
 
-    static member Extract (recordInfo : RecordTableInfo) (expr : Expr) =
-        extractQueryExpr recordInfo expr
+    static member Extract (recordInfo: RecordTableInfo) (expr: Expr) = extractQueryExpr recordInfo expr
```

Comment:
not consisten with the other functions, though 505 in the source is also not consistent

Reply:
I think this is all based on line length - it’s not necessarily going to be consistent

Reply:
as noted in the mainline text, the TaskSeq ruleset has a wider width and tends to work out ok IME

Reply:
will delte this